docs/support.mdx

```diff
@@ -1,111 +1,44 @@
-  </summary>
+To report bugs, make feature requests, and see what's coming in future releases, visit the individual GitHub repositories:
 
-For research papers, we encourage authors to acknowledge IBM Quantum using:
+[Qiskit SDK](https://github.com/Qiskit/qiskit)
 
->    We acknowledge the use of IBM Quantum services for this work. The views expressed are those of the authors, and do not reflect the official policy or position of IBM or the IBM Quantum team.
+[Qiskit Runtime](https://github.com/Qiskit/qiskit-ibm-runtime)
 
-Paper references should be cited as follows:
+[Qiskit Serverless](https://github.com/Qiskit/qiskit-serverless)
 
-> IBM Quantum. https://<span>quantum.ibm.com</span>/, 2021
+[Qiskit Transpiler Service](https://github.com/Qiskit/qiskit-ibm-transpiler)
 
-Quantum processing units in the paper should be referenced by their unique name (i.e., `ibmq_vigo`) and optionally
-adding the version (i.e., `ibmq_vigo` v1.0.2).  We also encourage referencing the processor.
-For example:
+[Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) - Click on the "Repo" link for a specific package to go to its GitHub repository
 
-> In this paper we used `ibmq_vigo`, which is one of the IBM Quantum Canary processors.
-
-An example of citing an IBM Quantum program:
-
-> IBM Quantum (2022). Estimator primitive (Version x.y.z) [computer software]. https://<span>quantum.ibm.com</span>/
-</details>
-
-<details>
-  <summary>
-    How do I cite IBM Quantum Composer in my research?
-  </summary>
-
-An example of citing IBM Quantum Composer:
-
-> IBM Quantum Composer. 2023. url: https://<span>quantum.ibm.com/composer</span>
-</details>
-
-<details>
-  <summary>
-    Why do I receive the error message `AttributeError: QuantumCircuit object has no attribute 'save_state'` when using `save_*`method on a circuit?
-  </summary>
-  
-The `save_*` instructions are part of Qiskit Aer project,
-a high performance simulator for quantum circuits. These instructions do not
-exist outside of Qiskit Aer and are added dynamically to the
-[QuantumCircuit](/api/qiskit/qiskit.circuit.QuantumCircuit#quantumcircuit-class) class by Qiskit Aer on import. If you would like to
-use these instructions you must first ensure that you have imported
-`qiskit_aer` in your program before trying to call these methods. You
-can refer to [qiskit_aer.library](https://qiskit.org/ecosystem/aer/apidocs/aer_library.html) for the details of these custom
-instructions included with Qiskit Aer.
-</details>
-
-<details>
-  <summary>
-    Why do my results from real devices differ from my results from the simulator?
-  </summary>
-
-The simulator models an ideal environment, without noise or decoherence. When jobs are run on the real devices, noise from the environment and decoherence cause the qubits to behave differently than in an ideal environment.
-</details>
-
-<details>
-  <summary>
-    How do I delete my account and data?
-  </summary>
+## Frequently asked questions
```

Comment:
Thoughts on this going above OS governance? I suspect most users care more about this.

Reply:
I'm good with moving it. Should check with @javabster -- what do you think Abby?

Reply:
Abby's good with it too